README.md

```diff
@@ -50,33 +50,13 @@ For command line help, visit our wiki page on Bitters’ [command line interface
   @import "base/base";
   ```
 
-5. When using Neat, uncomment the following line in `_base.scss`:
-
-  ```scss
-  @import "grid-settings";
-  ```
-
-  And import Neat after Bitters in your `application.css.scss` or main manifest file:
-
-  ```scss
-  @import "bourbon";
-  @import "base/base";
-  @import "neat";
-
-  // All other imports
-  ```
-
-  **Note:** If you are using Bitters and Neat without Ruby on Rails, you need to change the import for `neat-helpers` inside `_grid-settings.scss` to `"../neat/neat-helpers"`.
-
-  If you want to use Neat functions in Bitters, you can `@import "grid-settings";` before Neat, remove `@import "grid-settings";` from `_base.scss` and import the rest of bitters after. For example:
+5. Once Bourbon and Bitters are set up, you can begin to import your styles below them.
```

Comment:
4?

Alternatively, make all of the `1`’s and Markdown will build the order for you, on render.